src/smc-webapp/admin/users/ban.tsx

```diff
@@ -70,7 +70,7 @@ export class Ban extends Component<Props, State> {
     return (
       <ErrorDisplay
         error={this.state.error}
-        onClick={() => {
+        onClose={() => {
```

Comment:
there were a few cases, where `ErrorDisplay` had `onClick` inside of it. I bet this is a bug. I changed all of them like this one here